test-requirements.txt

```diff
@@ -1,3 +1,3 @@
-ansible==2.8.2
-molecule==2.20.1
-docker==4.0.2
+ansible==2.8.6
```

Comment:
Could you set this version as `min_ansible_version` in _meta/main.yml_ file? 

https://github.com/idealista/zookeeper_role/blob/fb3768bfae713a3ef53d5a571aec1cf4773dcc58/meta/main.yml#L6 (

Reply:
Done